pkg/sensors/tracing/kprobe_test.go

```diff
@@ -6970,3 +6970,67 @@ tetragon_missed_prog_probes_total{attach="wake_up_new_task",policy="__base__"} 0
 		prometheus.BuildFQName(consts.MetricsNamespace, "", "missed_prog_probes_total")))
 
 }
+
+func TestKprobeBpfCmd(t *testing.T) {
```

Comment:
So I guess the idea of the test is to detect the tetragon bpf program / map loading?

Reply:
Yes kept it small, but it can be used to detect all bpf commands.